Makefile

```diff
@@ -18,8 +21,20 @@ cwd_checked:=$(patsubst ./%,.%.shellchecked,${cwd})
 check: ${src_checked} ${tests_checked} ${cwd_checked}
 	echo OK
 
+flake8:
```

Comment:
do we want to touch `.shellchecked` files for this too (and maybe rename to `.checked` instead since its not always shellcheck)?

Also we ought to make the `check` target depend on this so `make check` does it too.

Reply:
The python generated files are more so you/pyc/__pycache__ which happen during interpretation.

:+1: to check executing flake8 target

Reply:
> The python generated files are more so you/pyc/pycache which happen during interpretation.

Sorry, I don't parse?

The `.checked` files are so we don't recheck things that are already checked and haven't been modified. I run `make check` every time I run cosa, so not rerunning it matters to me.

Reply:
I think I misunderstood your note. Is this more so to have `flake8` drop `.checked` files to keep `flake8` from rerunning it's checks when the `make flake8` is called?

Reply:
Yeah

Reply:
If the worry is the time it takes to check, I _think_ it should be fast enough. Here is a local run of `make flake8`:
```
real    0m0.333s
user    0m0.270s
sys     0m0.049s
```
If I run it on the entire directory it takes a bit longer:
```
real    0m0.342s
user    0m0.495s
sys     0m0.040s
```

I'm not against making such a change, but I'm not sure it's worth it _yet_.

Reply:
Could also start writing in a statically compiled language... :wink: